.circleci/config.yml

```diff
@@ -81,6 +81,46 @@ jobs:
           name: Execute retagger
           command: ./retagger -r ${REGISTRY} -o ${REGISTRY_ORGANISATION} -u ${REGISTRY_USERNAME} -p ${REGISTRY_PASSWORD}
 
+
+  e2eDockerhub:
+    environment:
+      REGISTRY: "docker.io"
+      REGISTRY_ORGANISATION: "giantswarm"
+    machine: true
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Set registry credentials
+          command: |
+            echo 'export REGISTRY_USERNAME=${DOCKERHUB_USERNAME}' >> $BASH_ENV
+            echo 'export REGISTRY_PASSWORD=${DOCKERHUB_PASSWORD}' >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Run e2e test
+          command: |
+            ./e2e.test -test.run ".*E2e.*" -test.v
+
+  retagDockerhub:
```

Comment:
Should we just drop quay completely? Or do we expect issues with that?

Reply:
I think a bunch of places are still hardcoded to use quay, so should probably keep it until there's been an organized move away